tests/testthat/test-checks.R

```diff
@@ -179,6 +179,54 @@ test_that("workflow objects", {
   })
 })
 
+test_that("workflow objects (will not tune, tidymodels/tune#548)", {
+  skip_if_not_installed("glmnet")
+
+  bare_rec <-
+    recipes::recipe(ridership ~ ., data = head(Chicago))
+
+  # don't warn when supplied tune args make sense given engine / steps
+  expect_silent(
```

Comment:
Maybe use `expect_warning(code, NA)` here until we have `expect_no_warning()` in testthat

Reply:
Addressed in [14faaa2](https://github.com/tidymodels/tune/pull/549/commits/14faaa2edc1897fc33ad1a3397ed315a9da519f6)!